src/WebJobs.Extensions.DurableTask.Analyzers/MethodInformation.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+
+namespace Microsoft.Azure.WebJobs.Extensions.DurableTask.Analyzers
+{
+    internal class MethodInformation
```

Comment:
It looks like this class is only used by `MethodAnalyzer`. If so, I suggest you make it a private inner class of `MethodAnalyzer` rather than making it a top-level class. One less file in the repo and it makes the expected usage more well-defined. :)